src/main/java/uk/gov/hmcts/reform/hmc/model/hearing/UnavailabilityType.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.hmcts.reform.hmc.model.hearing;
+
+public enum UnavailabilityType {
+    AMP,
```

Comment:
maybe just low English level in my side, but wouldn't be AM instead of AMP?

Reply:
This is existing code pulled over from civil-service repo into common place. not changing this here.